chord_metadata_service/chord/workflows/wdls/experiments_extended_json.wdl

```diff
@@ -0,0 +1,404 @@
+version 1.0
+
+workflow experiments_extended_json {
+    input {
+        String directory
+        File json_document
+        String drs_url
+        String katsu_url
+        String project_dataset
+        String access_token
+        Boolean validate_ssl
+    }
+
+    call prepare_download_list {
+        input:
+            json_document = json_document,
+            directory = directory
+    }
+
+    ## Calculate the size of the JSON document to determine if there are any URLs to download
+    Int urls_json_size = ceil(size(prepare_download_list.urls_json, "Gi") * 8000)
```

Comment:
I don't understand this line - why does it calculate the file size in gigabytes(?)

Reply:
No reason, I already changed